c7n/version.py

```diff
@@ -11,5 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Backed out in 09f6211958693d1a3296a03f651aea9efa1a70a4. Will look at empties as well ...